api/draft_registrations/serializers.py

```diff
@@ -156,6 +166,13 @@ def get_self_url(self, obj):
             },
         )
 
+    registration_schema = RegistrationSchemaRelationshipField(
+        related_view='schemas:registration-schema-detail',
+        related_view_kwargs={'schema_id': '<registration_schema._id>'},
+        required=False,
+        read_only=False,
+    )
```

Comment:
This one fits into the DRF MRO issue/update where `DraftRegistrationSerializer` and `DraftRegistrationDetailLegacySerializer` define a `RegistrationSchemaRelationshipField` with different params. The former with `required=True` and `read_only=False` while the latter uses the default (both False). It makes sense now why we want to explicit set both to `False` for `DraftRegistrationDetailSerializer` since it would use the one from `DraftRegistrationSerializer` due to DRF change.

Reply:
Confirmed that this is the only place we need to explicitly define `registration_schema`